languages/common-lisp/exercises/concept/leslies-lists/.docs/instructions.md

```diff
@@ -12,7 +12,7 @@ First thing is that Leslie needs to create a new empty list. A function called `
 (empty-list) ; => ()
 ```
 
-Oh no... Leslie actually has a few things in mind already so she needs a function that takes a three items (luckily Leslie never creates a list if she has less or more than three items) and creates a new shopping list with those things. Write a function called `list-of-things` which will take three items and makes a list of them.
+Oh no... Leslie actually has a few things in mind already so they needs a function that takes a three items (luckily Leslie never creates a list if they has less or more than three items) and creates a new shopping list with those things. Write a function called `list-of-things` which will take three items and makes a list of them.
```

Comment:
I think we'll need to change the verb conjugations after many of the swapped pronouns. E.g. "they needs" -> "they need"

Other than that, I think this is a good change :)

Reply:
@TheLostLambda please take another look I believe I have fixed the verb conjugations now.